colors.tdesktop-theme

```diff
@@ -150,7 +150,7 @@ dialogsNameFg: fg1; // chat list name text
 dialogsTextFg: fg4; // chat list message text
 dialogsTextFgService: bright_yellow; // chat list group sender name text (or media message type text)
 dialogsRippleBg: windowBgRipple; // chat list ripple effect (on mouse down)
-dialogsUnreadBg: bright_red; // chat list unread badge background for not muted chat
+dialogsUnreadBg: mix_orange; // chat list unread badge background for not muted chat
```

Comment:
I've tried the new unread message color and found it a bit dull in comparison to the old one. I understand, that the new version is more in line with the default theme, but I'm keeping the old color, sorry.